### PR TITLE
feat(cave): new page to Edit a Network

### PR DESCRIPTION
**Pull request by maximenathan**
Based on https://github.com/GrottoCenter/grottocenter-front/pull/56
A reprendre pour éditer un network plutot que pour créer un réseau, la page de création n'existera pas

- Basic Cave creation form
**API call doesn't work @Clm-Roig  any idea ?**

grotto

### CODE REVIEW

File: packages/web-app/src/components/appli/EntitiesForm/Cave/index.jsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      descriptionAndNameLanguage: {
+        language: data.language,
+        body: data.description,
+        title: data.descriptionTitle
+      },
+      depth: data.depth,
+      isDiving: data.isDiving,
+      length: data.length,
+      temperature: data.temperature
+    };
```
Comment (maximenathan) [inline packages/web-app/src/components/appli/EntitiesForm/Cave/index.jsx:129]: This is what we send to the api @Clm-Roig

**Comment by Clm-Roig** [comment]
I will make a PR on the API to improve this route, it's not perfect. I'm working on it now :slightly_smiling_face:

**Comment by maximenathan** [comment]
> I will make a PR on the API to improve this route, it's not perfect. I'm working on it now slightly_smiling_face

Should we be able to send multiple descriptions ?

**Comment by Clm-Roig** [comment]
> 
> Should we be able to send multiple descriptions ?

I don't think so. When an user create the cave, he just sends his own description no ?

**Comment by urien** [comment]
Oui il faut plusieurs descriptions.
Ce permet de découper par galerie, portion de la cavité.
Mais au si de décrire la géologie, l'hydrologie, la paléo, ....

Le dim. 28 nov. 2021 à 14:39, Clément Roig ***@***.***> a
écrit :

**Comment by Clm-Roig** [comment]
API route reworked (and it accepts multiple descriptions now) **available in production**: 
https://github.com/GrottoCenter/Grottocenter3/pull/670

JSON example (from the PR): 

```json 
{
  "depth": 42,
  "descriptions": [
    {
      "body": "There are many rocks.",
      "language": "eng",
      "title": "Geology"
    },
    {
      "body": "It's wet",
      "language": "eng",
      "title": "Hydrology"
    },
    {
      "body": "Very beautiful",
      "language": "eng",
      "title": "First gallery"
    }
  ],
  "documents": [
    109,
    161,
    191
  ],
  "isDiving": true,
  "latitude": 22.440979208779,
  "length": 530,
  "longitude": -83.980479240417,
  "massif": 4,
  "name": {
    "text": "La Cavité XYZ",
    "language": "eng"
  },
  "temperature": 8.36
}
```

**Comment by Clm-Roig** [comment]
Je ferme au profit de #460

**Pull request closed without merging**